test/api/test-draw.c

```diff
@@ -926,8 +927,8 @@ main (int argc, char **argv)
   hb_test_add (test_hb_draw_cff1_rline);
   hb_test_add (test_hb_draw_cff2);
   hb_test_add (test_hb_draw_ttf_parser_tests);
-  hb_test_add (test_hb_draw_font_kit_glyphs_tests);
-  hb_test_add (test_hb_draw_font_kit_variations_tests);
+if(0)  hb_test_add (test_hb_draw_font_kit_glyphs_tests);
+if(0)  hb_test_add (test_hb_draw_font_kit_variations_tests);
```

Comment:
@ebraminio any chance you can debug why these two tests fail?

Reply:
most likely libc's roundf used in _hb_iota is different with internal _hb_roundf this code used to enjoy.

Reply:
which

```
diff --git a/test/api/test-draw.c b/test/api/test-draw.c
index 3204acee0..fdf437f44 100644
--- a/test/api/test-draw.c
+++ b/test/api/test-draw.c
@@ -49,7 +49,7 @@ static void _hb_reverse (char *buf, unsigned int len)
 }
 static unsigned _hb_itoa (float fnum, char *buf)
 {
-  int32_t num = (int32_t) roundf (fnum);
+  int32_t num = (int32_t) floorf (fnum + .5f);
   unsigned int i = 0;
   hb_bool_t is_negative = num < 0;
   if (is_negative) num = -num;
@@ -934,8 +934,8 @@ main (int argc, char **argv)
   hb_test_add (test_hb_draw_cff1_rline);
   hb_test_add (test_hb_draw_cff2);
   hb_test_add (test_hb_draw_ttf_parser_tests);
-if(0)  hb_test_add (test_hb_draw_font_kit_glyphs_tests);
-if(0)  hb_test_add (test_hb_draw_font_kit_variations_tests);
+  hb_test_add (test_hb_draw_font_kit_glyphs_tests);
+  hb_test_add (test_hb_draw_font_kit_variations_tests);
   hb_test_add (test_hb_draw_estedad_vf);
   hb_test_add (test_hb_draw_stroking);
   hb_test_add (test_hb_draw_immutable);
```

makes it work